src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -123,4 +194,31 @@ public String getDisplayName() {
             return "Sealights CLI";
         }
     }
+
+    private CLIRunner createCLIRunner(CommandMode commandMode){
+        if(commandMode instanceof CommandMode.ConfigView){
+            CommandMode.ConfigView configView=(CommandMode.ConfigView) commandMode;
+            return new CLIRunner(configView.getBuildSessionId(),configView.getAppName(),configView.getBranchName(),
+                    configView.getBuildName(),configView.getAdditionalArguments(),null);
+        }else {
+            Properties properties = PropertiesUtils.toProperties(commandMode.getAdditionalArguments());
+            String appName = (properties.get("appname")!= null)? properties.get("appname").toString():null;
```

Comment:
get return Object appName is String.
when key doesn't  exist the method returns null